go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/drone-plugins/drone-sftp
 
-go 1.15
+go 1.18
```

Comment:
Were the Windows exec runners updated to golang 1.18? If not then this will just fail to build.